chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -39,127 +39,135 @@ namespace TemplateURLPrepopulateData {
 namespace {
 
 // Default order in which engines will appear in the UI.
-const std::vector<BravePrepopulatedEngineID> brave_engines_default = {
+constexpr BravePrepopulatedEngineID brave_engines_default[] = {
```

Comment:
these names should have `kBraveEnginesDefault` style now.

Reply:
Done.